src/tests/services/actions/permission-mode-action-creators.spec.ts

```diff
@@ -24,10 +24,10 @@ describe('Graph Explorer Permission mode creator\'', () => {
 
     it('creates CHANGE_PERMISSIONS_MODE_SUCCESS when changeMode is called with DISPLAY_DELEGATED_PERMISSIONS', () => {
```

Comment:
That comment should probably be changed to `'creates CHANGE_PERMISSIONS_MODE_SUCCESS when changeMode is called with PERMISSION_MODE_TYPE.User'`

```suggestion
    it('creates CHANGE_PERMISSIONS_MODE_SUCCESS when changeMode is called with PERMISSION_MODE_TYPE.User', () => {
```